src/UI/Component/Popover/ReplaceContentSignal.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace ILIAS\UI\Component\Popover;
+
+use ILIAS\UI\Component\Signal;
+
+/**
+ * This signal replaces the content of a Popover with some content returned by an ajax request.
```

Comment:
I agree, Signals used by several components should be moved into the `Component` folder. We can move this interface if the signal is also used by the Modal.